src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -575,8 +571,15 @@ protected void updateArmor()
         }
     }
 
-    private double getRandomPitch()
+    /**
+     * Damage per ranged attack.
+     *
+     * @return the attack damage
+     * Suppression because the method already explains the value.
+     */
+    @SuppressWarnings({"squid:S3400", "squid:S109"})
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S3400' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)